hm_pyhelper/miner_json_rpc/client.py

```diff
@@ -24,3 +24,7 @@ def get_peer_addr(self):
 
     def get_peer_book(self):
         return self.__fetch_data('peer_book', addr='self')
+
+    def get_firmware_version(self):
```

Comment:
I agree [with Aaron](https://github.com/NebraLtd/hm-pyhelper/pull/33#issuecomment-948962644), that this should be a convenience method that invokes the existing `get_summary` and returns the parsed firmware version.